index.js

```diff
@@ -210,10 +210,13 @@ var lastResorts = [
   ['BEER', 'Beerware'],
   ['BOOST', 'BSL-1.0'],
   ['BSD', 'BSD-2-Clause'],
+  ['CDDL', 'CDDL-1.1'],
```

Comment:
I just did a search on all packages on `npm` and only one single package has the license string `CDDL` (it was the package `node-fifo`). 4 packages have `CDDL-1.0` and 0 packages have `CDDL-1.1`. So I feel like this change might not be very helpful. (Let me know if my search was wrong.)

Maybe it would be better to think about the generic case: if you don't specify the version string (i.e `-1.1`) would we want to always suggest the newest version, regardless of the license? We could potentially work on this generic solution instead, but I'm not sure about the return on investment.